core/common/error_text.cpp

```diff
@@ -18,7 +18,7 @@ namespace fc::error_text {
       return kName;
     }
     std::string message(int value) const override {
-      return (const char *)nullptr;
+      return "zero category";
```

Comment:
```suggestion
      return static_cast<const char *>(nullptr);
```
revert this.  
`.message()` must return exact pointer passed to `_make_error_code`. i added `CategoryZero` completeness, most programs never use it (because nullptr is inaccessible).

Reply:
Zero Category deleted